flocker/apiclient/test/test_client.py

```diff
@@ -798,6 +815,21 @@ def test_logging(self, logger):
         return d
 
     @capture_logging(None)
+    def test_persistent_connections(self, logger):
+        """
+        ``FlockerClient`` can use a pool with persistent connections when
+        specified.
```

Comment:
:question: This is the bit that I'm slightly confused about. What does a pool of non-persistent connections even mean? 

And in my tests this morning it appeared that with `persistent_connections=True` my client established a new connection for each request and yet held it open....at least as far as I could see by running `netstat -antp` on client and server.
I may have been mistaken though.